source/shared/cpp/ObjectModel/Enums.cpp

```diff
@@ -220,6 +222,7 @@ namespace AdaptiveCards
             {CardElementType::Image, "Image"},
             {CardElementType::ImageSet, "ImageSet"},
             {CardElementType::Media, "Media"},
+			{CardElementType::MetaData, "MetaData"},
```

Comment:
MetaData is not a CardElement type.

Reply:
@jwoo-msft it is defined as a type here: https://adaptivecards.io/explorer/Metadata.html

Reply:
Removed